feat: LSDV-5242: MVP for dynamic LLM chat

robot-ci-heartex opened the pull request:
Hi @hlomzik!

This PR was created in response to a PRs in upstream repo:
- https://github.com/heartexlabs/label-studio-frontend/pull/1438

hlomzik committed:
[submodules] Build static heartexlabs/label-studio-frontend

hlomzik committed:
docs: LSF Update

hlomzik committed:
[submodules] Build static heartexlabs/label-studio-frontend

hlomzik committed:
Merge branch 'origin/develop' into 'fb-lsdv-5242'

hlomzik committed:
[submodules] Build static heartexlabs/label-studio-frontend

hlomzik committed:
Merge branch 'origin/develop' into 'fb-lsdv-5242'

hlomzik committed:
[submodules] Build static heartexlabs/label-studio-frontend

hlomzik committed:
docs: LSF Update

hlomzik committed:
Merge branch 'origin/develop' into 'fb-lsdv-5242'

nick-skriabin committed:
[submodules] Build static heartexlabs/label-studio-frontend

Workflow run: https://github.com/heartexlabs/label-studio/actions/runs/5524537879

nick-skriabin committed:
Merge branch 'origin/develop' into 'fb-lsdv-5242'

Workflow run: https://github.com/heartexlabs/label-studio/actions/runs/5524537879

nick-skriabin committed:
[submodules] Build static heartexlabs/label-studio-frontend

Workflow run: https://github.com/heartexlabs/label-studio/actions/runs/5525169313

nick-skriabin committed:
docs: LSF Update

Workflow run: https://github.com/heartexlabs/label-studio/actions/runs/5525169313

nick-skriabin committed:
Merge branch 'origin/develop' into 'fb-lsdv-5242'

nick-skriabin committed:
[submodules] Build static heartexlabs/label-studio-frontend

Workflow run: https://github.com/heartexlabs/label-studio/actions/runs/5525186680

nick-skriabin committed:
[submodules] Build static heartexlabs/label-studio-frontend

Workflow run: https://github.com/heartexlabs/label-studio/actions/runs/5574753413

nick-skriabin committed:
docs: LSF Update

Workflow run: https://github.com/heartexlabs/label-studio/actions/runs/5574753413

nick-skriabin committed:
Merge branch 'origin/develop' into 'fb-lsdv-5242'

Workflow run: https://github.com/heartexlabs/label-studio/actions/runs/5574753413

niklub commented:
/git merge develop
> Already up-to-date. Nothing to commit.
>
> [Workflow run](https://github.com/heartexlabs/label-studio/actions/runs/5575307848)

nick-skriabin committed:
[submodules] Build static heartexlabs/label-studio-frontend

Workflow run: https://github.com/heartexlabs/label-studio/actions/runs/5575851928

nick-skriabin committed:
[submodules] Build static heartexlabs/label-studio-frontend

Workflow run: https://github.com/heartexlabs/label-studio/actions/runs/5576612829

nick-skriabin committed:
[submodules] Build static heartexlabs/label-studio-frontend

Workflow run: https://github.com/heartexlabs/label-studio/actions/runs/5577154322

hlomzik committed:
[submodules] Build static heartexlabs/label-studio-frontend

Workflow run: https://github.com/heartexlabs/label-studio/actions/runs/5578118869
